widgets/common/src/filters/types.ts

```diff
@@ -17,8 +17,8 @@ enum LabelsOnlyRuleOperator {
 }
 
 enum AttributesOnlyRuleOperator {
-    Contain = 'contain',
-    NotContain = 'not_contain',
+    Contain = 'contains',
+    NotContain = 'not_contains',
```

Comment:
Optional:

Renaming the enum keys would keep them in line with the values, possibly reducing confusion

```suggestion
    Contains = 'contains',
    NotContains = 'not_contains',
```